lib/module.js

```diff
@@ -441,6 +441,8 @@ function stripBOM(content) {
   return content;
 }
 
+Module._stripBOM = stripBOM;
```

Comment:
@bnoordhuis agreed.  I exposed it as-is to avoid duplicating code - I'll create the new module and push a new commit up in a bit.